tests/CMakeLists.txt

```diff
@@ -37,8 +37,11 @@ add_test_case(mqtt_connection_connack_timeout)
 add_test_case(mqtt_connect_subscribe)
 add_test_case(mqtt_connect_subscribe_multi)
 add_test_case(mqtt_connect_unsubscribe)
+# we probably need to deprecate resubscribe or not
+# add_test_case(mqtt_connect_resubscribe) 
```

Comment:
Just delete the test

Reply:
If we are going to keep the resubscribe function (which I think we can), I'll build the test recently...

Reply:
test built